lib/elixir/src/elixir_expand.erl

```diff
@@ -1116,6 +1126,9 @@ format_error(unhandled_arrow_op) ->
   "unhandled operator ->";
 format_error(as_in_multi_alias_call) ->
   ":as option is not supported by multi-alias call";
+format_error({invalid_alias_module, Ref}) ->
+  io_lib:format("invalid alias module, expected an Elixir module, got: ~ts. To alias other BEAM languages modules, use the :as option",
```

Comment:
I would say something like "alias cannot be inferred automatically, please use the :as option"

Reply:
Good one, will update